### ISSUE TITLE
Cauldron recipes

### ISSUE THREAD

**Issue by Estrelaura** [MehVahdJukaar/amendments#172]
As discussed on discord, a request to add custom recipes to the cauldron or the ability to pull from other recipe types other than crafting (i.e. Farmer's Delight cooking).

**Comment by RonnymonnyTheProgrammer** [comment]
It would also be great if such recipes used Create's "mixing" folder, as most mods already support Create recipes.

**Comment by MehVahdJukaar** [comment]
Well of one has create might as well use their cauldron no?

**Comment by MehVahdJukaar** [comment]
Yeah that's my point. If they leave out create there's no use in using create own recipes as create itself is a there and those recipes aren't there either. Once I figure this out precisely I will add recipes, what theyw I'll be able to do depends on the system I already have and will be different very likely than other existing stuff create has

**Comment by MehVahdJukaar** [comment]
done
